cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -90,6 +100,14 @@ class TowerVariableValue(models.Model):
         ),
     ]
 
+    @api.depends("option_id", "variable_id")
+    def _compute_has_options(self):
+        """Determine if the variable has options."""
+        for record in self:
+            record.has_options = bool(record.variable_id.option_ids.ids)
+            if record.variable_id.option_ids.ids:
+                record.value_char = record.option_id.name
```

Comment:
You are modifying a non-computed field value in the compute method of another field.
If you want to compute a value for that field you should make that field computed instead. And add the corresponding dependencies.